oxd-client/src/main/java/org/gluu/oxd/client/ClientInterface.java

```diff
@@ -145,4 +145,9 @@
     @Consumes(MediaType.APPLICATION_JSON)
     GetJwksResponse getJwks(@HeaderParam("Authorization") String authorization, GetJwksParams params);
 
+    @POST
+    @Path("/get-op-discovery-config")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    GetOpDiscoveryConfigResponse getOpDiscoveryConfig(GetOpDiscoveryConfigParams params);
```

Comment:
rename it everywhere to just `/get-discovery` -> `GetDiscoveryResponse`

Reply:
Changes done.